src/styles/blocks/catalog.scss

```diff
@@ -5,21 +5,23 @@
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
+  gap: 48px;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: $duraction;
```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
just list all params witch will be changed